src/aboutPage.js

```diff
@@ -0,0 +1,18 @@
+import htmlTag from './htmlTags';
+
+export default function aboutPage() {
```

Comment:
- [ ] It uses ES6 syntax
Let's double check all the functions and convert it to arrow functions.
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png)

Reply:
Please be more specific:
For exporting I used literal function syntax to take advantage of the `export default` keywords;